{
  "path": "/documentation/foundation/urlauthenticationchallenge/1413033-error",
  "type": "Instance Property",
  "name": "error",
  "desc": "The error object representing the last authentication failure.",
  "items": [],
  "declaration": "var error: Error? { get }"
}